modules/installation-creating-aws-worker.adoc

```diff
@@ -180,11 +180,11 @@ the CloudFormation template for the security group and roles.
 section of this topic and save it as a YAML file on your computer. This template
 describes the networking objects and load balancers that your cluster requires.
 
-. If you specified an `m5` instance type as the value for `WorkerInstanceType`,
-add that instance type to the `WorkerInstanceType.AllowedValues` parameter
-in the CloudFormation template.
+. Optional: If you specified an `m5` instance type as the value for `WorkerInstanceType`, add that instance type to the `WorkerInstanceType.AllowedValues` parameter in the CloudFormation template.
```

Comment:
The allowed values had been removed by https://github.com/openshift/installer/commit/5db41b0a5055c1a98c92f8899c4689ec75cf4bc8, I think we can remove this `Optional` part.

Reply:
Oh, I noticed `Allowed values` are still in the document, looks like we need to change lots of things.
So please ignore my above comment, and let's create a separate PR to fix this. @bscott-rh wdyt?

Reply:
Started a thread in installer Slack to discuss this.